provisioner/terraform/install_test.go

```diff
@@ -101,8 +136,42 @@ func mainJSON(versions ...*version.Version) product {
 	return mj
 }
 
-func exeContent(v *version.Version) []byte {
-	return []byte(strings.ReplaceAll(terraformExecutableTemplate, "${ver}", v.String()))
+func mustCreateZips(t *testing.T, tmpDir string, v *version.Version) {
```

Comment:
I think you can split this function into multiple shorter ones for readability purposes:

`createZipForPlatform`, `generateExecutableContent`, `writeExecutableToZip`, `writeLicenseToZip`, etc. I think you know what I mean :)